andela/thorin-cfh
# #156882146 Configure ESLint Linter

#### What does this PR do?
Configure ESlinter to ensure the codebase adheres strictly to the airbnb style guide.
#### Description of Task to be completed?
N/A
#### How should this be manually tested?
Clone the repo and install application dependencies. Run command  `./node_modules/.bin/eslint  <filename>`. You should see lint errors.
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#156882146
#### Screenshots (if appropriate)
N/A
#### Questions: N/A
